cl/lib/elasticsearch_utils.py

```diff
@@ -1654,6 +1675,68 @@ def merge_unavailable_fields_on_parent_document(
                     result["plain_text"] = recap_docs_dict.get(
                         result["id"], ""
                     )
+
+        case SEARCH_TYPES.OPINION if request_type == "api" and not highlight:
+            # Retrieves the Opinion plain_text from the DB to fill the snippet
+            # when highlighting is disabled. Considering the same prioritization
+            # as in the OpinionDocument indexing into ES.
+
+            opinion_ids = {
+                doc["_source"]["id"]
+                for entry in results
+                for doc in entry["child_docs"]
+            }
+            opinions = (
+                Opinion.objects.filter(pk__in=opinion_ids)
+                .annotate(
+                    text_to_show=Case(
+                        When(
+                            ~QObject(html_columbia__exact=""),
```

Comment:
We don't need to add the `__exact` lookup. According to the documentation, it is assumed to be exact if you don’t provide a lookup type.

![image](https://github.com/freelawproject/courtlistener/assets/55959657/c8f3ac64-8c2d-4ae1-99e6-ea2cdc1774c0)

Reply:
thanks I've removed `__exact` from the lookups